tokio-util/Cargo.toml

```diff
@@ -18,7 +18,7 @@ categories = ["asynchronous"]
 
 [features]
 # No features on by default
-default = []
+default = ["tokio"]
```

Comment:
I think all other features also need to depend on the tokio feature.

Reply:
That's right, fixing!

Reply:
Hm, using `tokio` as a feature flag is not the best approach either since it gets activated with other flags such as `tokio/net`. I'm gonna rework the feature combo to make it more consistent else every other feature will simply have to depend on `tokio/sync` as it's currently implemented.